tests/algorithms/test_bc.py

```diff
@@ -212,6 +212,61 @@ def test_that_bc_improves_rewards(
     )
 
 
+def test_gradient_accumulation(
+    cartpole_venv: vec_env.VecEnv,
+    rng,
+    pytestconfig,
+):
+    demonstrations = make_expert_transition_loader(
+        cache_dir=pytestconfig.cache.makedir("experts"),
+        batch_size=6,
+        expert_data_type="transitions",
+        env_name="seals/CartPole-v0",
+        rng=None,
+        num_trajectories=17,
```

Comment:
Is it important this is `17`? If not, I'd suggest adding a `cartpole_expert_transitions` fixxture to `tests/algorithms/conftest.py` factoring out what's currently embedded inside `cartpole_bc_trainer`.

Reply:
I wanted a number that didn't divide cleanly into the minibatch size, but also didn't leave any remainders of 1, because an incorrect implementation can accidentally work for batch sizes of 1

Reply:
Gotcha, this makes sense :)

Reply:
But feel free to refactor it. Pytest fixtures are fiddly and I didn't want to put the work into dealing with that when I was just getting the tests to work.